TaskManagement/src/TaskManagment/DB methods/Queries.java

```diff
@@ -0,0 +1,8 @@
+package TaskManagerProject;
```

Comment:
The package name "TaskManagerProject" should follow the Java naming conventions, which suggest using lowercase letters for package names to differentiate them from class names. Consider renaming it to "taskmanagerproject" or something similar.